nextflow_schema.json

```diff
@@ -155,6 +156,11 @@
                 "star_ignore_sjdbgtf": {
                     "type": "string",
                     "description": "Ignore the SJDB GTF file."
+                },
+                "seq_center": {
+                    "type": "string",
+                    "description": "Name of sequencing center for BAM read group tag.",
+                    "default": null
```

Comment:
Maybe it would be better to include this in the samplesheet? This could be different on a per-sample basis.

Reply:
The `seq_center` parameter already existed in the workflow before this request, I'm just making it available via the schema in this PR. I think that moving this to a samplesheet column is a fair idea, but perhaps it can go in a separate PR. Is that reasonable?

Reply:
Absolutely. Created https://github.com/nf-core/scrnaseq/issues/150.